cli/src/main/package/setup.bat

```diff
@@ -6,41 +6,8 @@ Set _fBRed=[91m
 Set _RESET=[0m
 
 pushd %~dp0
-echo Setting up IDEasy in %CD%
+echo Setting up IDEasy from %CD%
 
-REM https://stackoverflow.com/questions/61888625/what-is-f-in-the-for-loop-command
-for %%f in (ideasy.exe) do set "p=%%~$PATH:f"
-if not defined p (
-  for /F "tokens=2* delims= " %%f IN ('reg query HKCU\Environment /v PATH ^| findstr /i path') do set USER_PATH=%%g
```

Comment:
It looks like the setup.bat is overwriting the whole `USER_PATH`.
Maybe re-add this.

Reply:
It seems you managed to reproduce this bug as the first developer in our team:
* https://github.com/devonfw/ide/issues/954
* https://github.com/devonfw/ide/issues/1071

It would be very helpful to analyse the root cause of this bug.
Your "suggested solution" is to re-add the check if `ideasy.exe` is already on the PATH.
I remove this on purpose because we changed the way we install IDEasy compared to previous releases and the upgrade will then fail to make the newly installed ideasy.exe to be found first on the PATH.
IMHO we should rather fix the real bug here.
However, we should also test that if we run the new installer multiple times, we do not keep adding duplicated entries to the PATH what might indeed be a bug in my code here.

Reply:
Done. Thanks for finding that there is a bug.
I did a severe copy & paste bug using `IDE_ROOT` instead of `PATH` for reading the USER_PATH.
Fixed.